bentoml/_internal/frameworks/transformers.py

```diff
@@ -1,86 +1,41 @@
-import re
+from __future__ import annotations
+
+import os
 import typing as t
 import logging
 import importlib.util
```

Comment:
lets also do
```python
import importlib
```
above this line for robust.

Reply:
Added.